specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/examples/createSubscription.json

```diff
@@ -4,14 +4,13 @@
     "invoiceSectionName": "b859d705-1ed7-4769-9d4b-19238e9e8ac2_10_31",
     "api-version": "2018-11-01-preview",
     "body": {
-      "displayName": "Test Ea Azure Sub",
+      "displayName": "Contoso MCA subscription",
       "billingProfileId": "/providers/Microsoft.Billing/billingAccounts/12a6eed9-5d39-4103-a5db-2db1db0a8c67/billingProfiles/dbcb81c8-cda9-4b01-b9fc-831e6a2e152e",
       "skuId": "0001",
-      "costCenter": "myCostCenter123",
+      "costCenter": "135366376",
       "owner": {
         "objectId": "973034ff-acb7-409c-b731-e789672c7b31"
-      },
-      "additionalParameters": {}
```

Comment:
is this deletion by intention?

Reply:
This is just an example and right now we are not using additional parameters in our code